README.md

```diff
@@ -5,7 +5,7 @@
 
 The [GitHub Enterprise Importer](https://docs.github.com/en/migrations/using-github-enterprise-importer) (GEI, formerly Octoshift) is a highly customizable API-first migration offering designed to help you move your enterprise to GitHub Enterprise Cloud. The GEI-CLI wraps the GEI APIs as a cross-platform console application to simplify customizing your migration experience.
 
-> GEI is in a public beta for GitHub Enterprise Cloud.
+> GEI is generally available for repository migrations originating from Azure DevOps or GitHub that target GitHub Enterprise Cloud. 
```

Comment:
The nit of all nits!  Remove trailing newline :grin: 

```suggestion
> GEI is generally available for repository migrations originating from Azure DevOps or GitHub that target GitHub Enterprise Cloud.
```

Reply:
🏅